### ISSUE TITLE
"Uploaded" alert shows on upload cancel

### ISSUE THREAD

**Issue by faisalnjs** [faisalnjs/Virtual-Checker#4]
When closing the upload page popup, the upload success alert displays even if no assets were uploaded.

![image](https://github.com/user-attachments/assets/fcf00309-6b58-4bcf-9053-0df59dca702e)

**Comment by faisalnjs** [comment]
Cannot differentiate between window close (by user, cancel) and window close (by self, success).

```js
const newWindow = window.open(url, '_blank', windowFeatures);
const checkWindowClosed = setInterval(function () {
  if (newWindow && newWindow.closed) {
    clearInterval(checkWindowClosed);
    ui.modeless(`<i class="bi bi-cloud-upload"></i>`, "Uploaded");
    init();
  }
}, 1000);
```

**Comment by faisalnjs** [comment]
Fixed: 61c99aab12170d336d3f070f35228a697f21c8b1
